app/uk/gov/hmrc/hec/controllers/TaxCheckController.scala

```diff
@@ -36,11 +38,25 @@ class TaxCheckController @Inject() (
   taxCheckService: TaxCheckService,
   authenticateGG: GGAuthenticateAction,
   authenticatedGGOrStride: GGOrStrideAuthenticateAction,
+  config: Configuration,
+  auth: BackendAuthComponents,
   cc: ControllerComponents
 )(implicit ec: ExecutionContext)
     extends BackendController(cc)
     with Logging {
 
+  val internalAuthEnabled: Boolean = config.get[Boolean]("internal-auth.enabled")
+
+  val permission: Predicate.Permission = Predicate.Permission(
```

Comment:
maybe something like `internalAuthPermission` would make it clearer that this is definitely to do with internal-auth

Reply:
[updated](https://github.com/hmrc/hec/pull/55/commits/3dd88d65dfd4c7df63b233da1cc9608d0ee25a1d)

Reply:
I meant updating what is called `permission` now to `internalAuthPermission` - `internalAuthEnabled` seemed like a pretty good name already :smile:

Reply:
oh, my bad 🤦‍♀️